user.py

```diff
@@ -51,10 +51,8 @@ def profile(cls):
         """
         User profile
         """
-        if request.method == "GET" and request.is_xhr:
-            user, = cls.browse([request.nereid_user.id])
-            return jsonify(user.serialize())
-        return super(NereidUser, cls).profile()
+        user, = cls.browse([request.nereid_user.id])
+        return jsonify(user.serialize())
```

Comment:
This is a bad idea. Assumes that nereid-project wont be used for example with nereid-webshop